java/buildconf/ivy/ivy-suse.xml

```diff
@@ -76,6 +76,7 @@
         <dependency org="suse" name="redstone-xmlrpc" rev="1.1_20071120" />
         <dependency org="suse" name="redstone-xmlrpc-client" rev="1.1_20071120" />
         <dependency org="suse" name="salt-netapi-client" rev="0.21.0" />
+        <dependency org="suse" name="saxon9" rev="9.4.0.7" />
```

Comment:
Introducing saxon raises the question of xalan: do we still want to depend on it? Can we get rid of it?